### DISCUSSION TITLE
Enable to push to other location than github.com

### DISCUSSION THREAD

**Discussion by jeising** [JamesIves/github-pages-deploy-action#621] [Ideas]
Thanks for this great action! Do you think it would be feasible or maybe it is already possible to define a different target address than github.com? This could make it possible to use this action in a self hosted github instance.

If I am not mistaken this might be the place, which would need to be adjusted:
https://github.com/crazy-max/ghaction-github-pages/blob/2e4f8c2395da12757455b259f9789e5c74778d1c/src/main.ts#L38

**Comment by JamesIves** [comment]
I think that's a reasonable request as GitHub Actions is supported by Enterprise now. The following PR is now available for review if you'd like to try it out: https://github.com/JamesIves/github-pages-deploy-action/pull/622, you can also point your workflow to `releases/v4-beta` if you have an enterprise instance to test it in. 

PS the url you linked isn't part of this project 😉

**Reply by jeising** [reply]
Ups ... I guess I was searching to freely 😄

Thanks for the change! 
It works like a charm! Admit it: You are the Flash! 🥇 🥇 🥇

**Reply by JamesIves** [reply]
Glad it worked! This has been releases in version `4.1.0`, you can point your workflow to `JamesIves/github-pages-deploy-action@4.1.0`

**Reply by jeising** [reply]
Awesome!
